hca/dss/util/__init__.py

```diff
@@ -58,7 +58,13 @@ def hardlink(source, link_name):
         if source_stat.st_dev != dest_stat.st_dev or source_stat.st_ino != dest_stat.st_ino:
             raise
     except OSError as e:
-        if e.errno == errno.EMLINK:
+        # See https://docs.python.org/3/library/errno.html
```

Comment:
```suggestion
```

Redundant, any Python programmer can locate the documentation for the standard library. We don't link to `logging`, `shutil` etc. and `errno` is no different.

Reply:
Comment removed